admin.html

```diff
@@ -19,9 +20,13 @@
     <div class ="navbar">
       <p class="title">RPI Shuttle Tracker</p>
       <a class ="logout button" href="/admin/logout/">Logout</a>
-      <div class ="button logout" onclick="Admin.showMapPanel();">Create Route</div>
+      <div class ="button logout" onclick="Admin.toggleDisplay();">Toggle Route Display</div>
+      <div class ="button logout rtb" onclick="Admin.showMapPanel();">Create Route</div>
```

Comment:
All of the element attributes should have no spacing around the `=`, i.e. `<div class="button...">`